frontend/src/components/recruiterSignUp/StepTwoRecruiterSignUp.js

```diff
@@ -2,55 +2,63 @@ import React, { useState } from 'react'
                             </Form.Group>
 
-                            <div className="company mb-3">
-                                <Form.Group className="mt-3 mb-3" controlId="formGridCompany">
-                                        <Form.Label><span style={{fontSize: '15px', color: '#777'}}>Company Name</span></Form.Label>
-                                        <Form.Control value={company} id="company" onChange={e => setValue(e)} />
-                                </Form.Group>
-
-                                <Form.Group className="mt-3 mb-3" controlId="formGridDescription">
-                                    <Form.Label><span style={{fontSize: '15px', color: '#777'}}>Short Description</span></Form.Label>
-                                    <Form.Control as="textarea" value={description} id="description" onChange={e => setValue(e)} />
-                                </Form.Group>
-                            </div>
-                            
+                            <Form.Group className="mt-3 mb-3" controlId="formGridDescription">
+                                <Form.Label>Short Description</Form.Label>
+                                <Form.Control required as="textarea" value={description} id="description" minlength="1" maxLength="500" onChange={e => setValue(e)} />
+                                <Form.Control.Feedback type="invalid">
+                                        Please enter a short description about your company.
+                                </Form.Control.Feedback>
+                            </Form.Group>
                             <Form.Group className="mb-4 mt-4" controlId="formGridLogo">
-                                <Form.Label>Link Company Logo</Form.Label>
-                                <Form.Control className="mb-2" type="url" id="logoLink" value={logoLink} onChange={e => setValue(e)} />
+                                <Form.Label>Company Logo Link</Form.Label>
+                                <Form.Control required className="mb-2" type="url" id="logoLink" pattern="(https?:\/\/.*\.(?:png|jpg))" value={logoLink} onChange={e => setValue(e)} />
+                                <Form.Control.Feedback type="invalid">
+                                        Please enter a valid link to your company logo.
+                                </Form.Control.Feedback>
                             </Form.Group>
 
                             <Form.Group className="mb-4 mt-4" controlId="formGridWebsite">
                                 <Form.Label>Company Website</Form.Label>
-                                <Form.Control className="mb-2" type="url" id="website" value={website} onChange={e => setValue(e)} />
+                                <Form.Control required className="mb-2" type="url" id="website" value={website} onChange={e => setValue(e)} />
+                                <Form.Control.Feedback type="invalid">
+                                        Please enter a valid email.
```

Comment:
Asks for a valid email when it's a field for company website

Reply:
done, changed to "Please enter a valid link to your company website."